SOP/SOP-operator.adoc

```diff
@@ -0,0 +1,79 @@
+ifdef::env-github[]
+:status:
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:table-caption!:
+endif::[]
+
+:toc:
+:toc-placement!:
+
+= Mobile Developer Console Operator- Standard Operating Procedures
+
+:toc:
+toc::[]
+
+== Overview
+
+The following guide outlines the steps required to manage and solve issues in the https://github.com/aerogear/mobile-developer-console-operator[Mobile Developer Console Operator].
+
+== Reference Articles
+
+- https://github.com/aerogear/mobile-developer-console[Mobile Developer Console]
+- https://github.com/aerogear/mobile-developer-console-operator[Mobile Developer Console Operator]
+- https://prometheus.io/docs/practices/alerting/[Prometheus Alerts documentation]
+
+== Success Indicators
+
+All alerts should appear as green in the Prometheus Alert Monitoring.
+
+== Critical 
+
+=== MobileDeveloperConsoleOperatorDown
+
+. Switch to the Mobile Developer Console namespace by running `oc project <namespace>`  E.g `oc project mobile-developer-console`.
+. Check if the operator pod to is present by running `oc get pods | grep operator`.
+
++
+[source,shell]
+----
+$ oc get pods | grep operator
+mobile-developer-console-operator-58c8877fd8-g6dvr          1/1       Running   3          9d
+----
+
+If the deployment is present, attempt to scale up the operator pod by running `oc `
+
+=== Warning
+
+==== MobilDeveloperConsolePodCPUHigh
+
+This alert indicates that the Service pod(s) is/are using more CPU than expected.
+
+. Please following the <<To capture the logs>> procedure in order to capture the required information to send it to its maintainers.
+. Following the steps <<To scale the pod>> in order to try to solve performance issues.
+
+==== MobilDeveloperConsolePodMemoryHigh
+
+This alert indicates that the Service pod(s) is/are using more Memory than expected.
+
+. Please following the <<To capture the logs>> procedure in order to capture the required information to send it to its maintainers.
+. Following the steps <<To scale the pod>> in order to try to solve performance issues.
+
+
+=== To capture the logs
+
+. Capture a snapshot of the 'Mobile Developer Console Application' Grafana dashboard and track it over time. The metrics can be useful for identifying performance issues over time.
+. Capture the Service logs
+.. Get the pod names by running `oc get pods`
+.. Save the logs by running `oc logs <mdc-podname> > <filename>.log` for any pod
+. Capture CR Status by running `oc describe MobileDeveloperConsole`.
+
+=== To scale the pod
+
+. Scale the Mobile Develper Console pod
+.. Get the replication controller name by running `oc get rc | grep mdc`
+.. Check the number of DESIRED and CURRENT replicas
```

Comment:
```suggestion
.. Check the number of DESIRED and CURRENT replicas, and make sure that they're the same:

\```
oc get deploymentconfig --selector='app=mdc'
NAME      REVISION   DESIRED   CURRENT   TRIGGERED BY
mdc       3          1         1         config,image(mdc-imagestream:1.1.4),image(mdc-oauth-proxy-imagestream:latest)
\```

If they're not, this may indicate a problem with the ability to scale up (capacity, missing images, etc), or the ability to scale down (a node where a pod was scheduled might be in trouble or something).
```

^ if accepting this suggestion, remove the escapes for the code block delimiters -- I had to put them in since the `suggestion` feature also uses those.